requirements.txt

```diff
@@ -1,10 +1,12 @@
-app-common-python==0.2.6
-boto3==1.29.7
-insights-core==3.2.24
-git+https://github.com/RedHatInsights/insights-core-messaging@1.2.4
-jsonschema==4.20.0
-python-json-logger==2.0.7
-requests==2.31.0
-sentry-sdk==1.37.1
-watchtower==3.0.1
-prometheus_client==0.19.0
+app-common-python>=0.2.3
+boto3<=1.28.64
+confluent-kafka>=2.0.0
+insights-core>=3.1.2
+insights-core-messaging>=1.2.4
+jsonschema>=4.0.0
+prometheus-client>=0.16.0
+python-json-logger
+PyYAML>=6.0
+requests
```

Comment:
Same as in the other file, which version of `requests`?

Reply:
Done